tests/epyccel/recognised_functions/test_numpy_types.py

```diff
@@ -12,6 +12,13 @@
 from pyccel.decorators import types, template
 from pyccel.epyccel import epyccel
 
+if sys.platform == 'win32':
+    RTOL = 1e-13
+    ATOL = 1e-14
+else:
+    RTOL = 2e-14
+    ATOL = 1e-15
```

Comment:
We do not need to treat Windows differently unless it is really needed. Here we are only adding two numbers, so hopefully we can use the same (smaller) tolerance for all operating systems.

Reply:
@rakati I understand your suggestion, but I would like to point out that the types used in this situation are numpy.float64 and numpy.complex128. Therefore, we only need to use one tolerance value for both types. It is not necessary to have separate tolerance values for 32 precision in this case.

Reply:
All right, I suggest naming it `RTOL_64`. If you already found a suitable one for 32 bits too, you can add it for future use.